apps/funke/app/_layout.tsx

```diff
@@ -0,0 +1,163 @@
+import type { OpenId4VcHolderAppAgent } from '@package/agent'
+
+import { AgentProvider } from '@package/agent'
+import { useTransparentNavigationBar, Provider, NoInternetToastProvider, isAndroid } from '@package/app'
+import { Heading, Page, Paragraph, XStack, YStack, config, useToastController } from '@package/ui'
+import { getSecureWalletKey } from '@package/utils'
+import { DefaultTheme, ThemeProvider } from '@react-navigation/native'
+import { useFonts } from 'expo-font'
+import { Stack } from 'expo-router'
+import * as SplashScreen from 'expo-splash-screen'
+import { useEffect, useState } from 'react'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+
+import { DeeplinkHandler } from './utils'
+
+import { initializeAppAgent } from '.'
+
+void SplashScreen.preventAutoHideAsync()
+
+export const unstable_settings = {
+  // Ensure any route can link back to `/`
+  initialRouteName: 'index',
+}
+
+export default function HomeLayout() {
```

Comment:
Still feel we can improve on duplication here (e.g. with fonts, initialization, hooks). But let's merge now and we can look at it later if needed. Good chance we can all throw it away in a few months anyway 🤷